packages/node-client/lib/index.ts

```diff
@@ -273,15 +273,17 @@ export class Nango {
      * @param connectionId - This is the unique connection identifier used to identify this connection
      * @param forceRefresh - Optional. When set to true, this obtains a new access token from the provider before the current token has expired
      * @param refreshToken - Optional. When set to true, this returns the refresh token as part of the response
+     * @param tokenName - Optional. Pass in a secondary token name to ensure the token is refreshed and is retrieved
      * @returns A promise that resolves with a connection object
      */
     public async getConnection(
         providerConfigKey: string,
         connectionId: string,
         forceRefresh?: boolean,
-        refreshToken?: boolean
+        refreshToken?: boolean,
+        tokenName?: string
```

Comment:
should the type be 'jwtToken' instead of string?

Reply:
Well, right now that is the only option, but this could be extended to return the user credential from github-app-oauth, or even the slack user token

Reply:
That's fair – if anything we've learned the more strict and less footguns the better especially wrt the public api and sdk.

Updated with https://github.com/NangoHQ/nango/pull/4611/commits/29af3c5ea1e6226bdd69c09afe53f4c20a3b054a